frontend/src/pages/AccessRequestPage.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { useEffect } from 'react';
+import './AccessRequestPage.scss';
+import { Container, Row, Col, ButtonToolbar, Button } from 'react-bootstrap';
+import { ILookupCode } from 'actions/lookupActions';
+import {
+  getCurrentAccessRequestAction,
+  getSubmitAccessRequestAction,
+  toAccessRequest,
+} from 'actionCreators/accessRequestActionCreator';
+import { IAccessRequestState } from 'reducers/accessRequestReducer';
+import { IUserInfo, IAccessRequest } from 'interfaces/accessRequests';
+import { Formik } from 'formik';
+import { Form, Input, TextArea, Select, SelectOption } from '../components/common/form';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from 'reducers/rootReducer';
+import { ILookupCodeState } from 'reducers/lookupCodeReducer';
+import * as API from 'constants/API';
+import * as actionTypes from 'constants/actionTypes';
+import _ from 'lodash';
+import { AccessRequestSchema } from 'utils/YupSchema';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import { IGenericNetworkAction } from 'actions/genericActions';
+
+interface IAccessRequestForm extends IAccessRequest {
+  agency: number;
+  role: string;
+}
+
+/**
+ * The AccessRequestPage provides a way to new authenticated users to submit a request
+ * that associates them with a specific agency and a role within the agency.
+ * If they have an active access request already submitted, it will allow them to update it until it has been approved or disabled.
+ * If their prior request was disabled they will then be able to submit a new request.
+ */
+const AccessRequestPage = () => {
+  const keycloakWrapper = useKeycloakWrapper();
+  const keycloak = keycloakWrapper.obj;
+  let userInfo = keycloak?.userInfo as IUserInfo;
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getCurrentAccessRequestAction());
+  }, []);
+
+  const data = useSelector<RootState, IAccessRequestState>(
+    state => state.accessRequest as IAccessRequestState,
+  );
+  const lookupCodes = useSelector<RootState, ILookupCode[]>(
+    state => (state.lookupCode as ILookupCodeState).lookupCodes,
+  );
+  const request = useSelector<RootState, IGenericNetworkAction>(
+    state => (state.network as any)[actionTypes.ADD_REQUEST_ACCESS] as IGenericNetworkAction,
+  );
+  const agencies = _.filter(lookupCodes, (lookupCode: ILookupCode) => {
+    return lookupCode.type === API.AGENCY_CODE_SET_NAME;
+  });
+  const roles = _.filter(lookupCodes, (lookupCode: ILookupCode) => {
+    return lookupCode.type === API.ROLE_CODE_SET_NAME;
+  });
+
+  let checkAgencies: {} | null | undefined;
```

Comment:
Seems like you could just use: *any* here.

Reply:
done